providers/certification-client/units/client-cert-desktop-18-04.pxu

```diff
@@ -132,6 +133,7 @@ nested_part:
     # with.
     power-management-reboot-poweroff-cert-automated
     tpm2.0_3.0.4
+    acpi_oem_osi_strings
```

Comment:
```suggestion
    acpi-automated
```

I think this should be placed higher in the list to avoid being launched after the device was suspended (maybe after `info-attachment-cert-automated`?).